### ISSUE TITLE
Export methods for the Charts

### ISSUE THREAD

**Issue by tsvetomir** [telerik/kendo-angular#357]
The charts will expose the following methods:

* exportPDF
* exportImage
* exportSVG

These can be used in conjunction with the [File Saver](http://www.telerik.com/kendo-angular-ui/components/filesaver/) utility to export the chart content to a file.

**Comment by tsvetomir** [comment]
We decided to remove the exportPDF method as it massively increases the script size.

Producing PDFs is still easy by calling exportPDF from the kendo-drawing package:
```ts
const visual = this.chart.exportVisual();
exportPDF(visual, {
  paperSize: "A4",
  landscape: true
}).then((dataURI) => {
  saveAs(dataURI, 'chart.pdf');
});
```

**Comment by tsvetomir** [comment]
The methods are implemented in the v0.19.0 version of the **@progress/kendo-angular-charts** package. Documentation will go live with a site update tomorrow.
